### PR TITLE
added h_bc function

### PR DESCRIPTION
**Pull request by nghayes**
Coded h_bc function using general boundary conditions h_0 and h_1.

### CODE REVIEW

File: 1D/matlab/wave_no_eb.m [hunk 1]
```diff
@@ -57,8 +57,14 @@
 
 function u = h_bc(x,t)
 % This function returns u at x and t.
-% x is supposed to be on the boundary    
-   % NH 
+% x is supposed to be on the boundary
+% h_0, h_1 are our boundary conditions
+if x == 0
```
Comment (appelo) [inline 1D/matlab/wave_no_eb.m]: @nghayes  Looks good, could be a touch dangerous to test on equality for double precision numbers. Maybe test if x < L/2 instead? 

The use of h_1 and h_0 is good! How about also defining those further down the file?

Comment (nghayes) [reply 1D/matlab/wave_no_eb.m]: I updated the code and pushed again (not sure if I did it right?).

**Review by appelo — APPROVED** [review]
@nghayes 

Awesome!
